src/DotNetProjectFile.Analyzers/MsBuild/Authors.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace DotNetProjectFile.MsBuild;
 
-public sealed class Authors : Node
+public sealed class Authors : Node<string>
 {
     public Authors(XElement element, Node parent, Project project) : base(element, parent, project) { }
 
-    public string Value => Element.Value;
+    public override string? Value => Element.Value;
```

Comment:
Since `Element.Value` is not nullable, the `?` can be dropped. This still fits with the nullable contract of the parent. (Same applies for all other Node<string> implementations).

Reply:
For some reason, I was certain that it was nullable. Anyway, with the default implementation this is not longer relevant.